.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = contrib/resource_pool
 	url = https://github.com/elsid/resource_pool.git
 	branch = master
+[submodule "contrib/nlohmann_json"]
```

Comment:
Is it better to get `nlohmann_json` as an external project and check out it only for benchmarks target in the same way as `gtest`?

Reply:
Done